src/components/FullNameInput.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import * as React from 'react';
+import {StyleSheet, View} from 'react-native';
+import StringInput from './StringInput';
+import TitlePicker from './TitlePicker';
+import type {User} from '../types.js';
+
+const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+  },
+});
+
+const FullNameInput = ({
+  user,
```

Comment:
It's about Component Design. FullNameInput don't need to know user object. It only needs to know First name, Last name, Title object. and function writeValue should be function onChangeLocation

Reply:
I have rewritten the component accordingly.